src/authz/index.ts

```diff
@@ -101,15 +101,17 @@ export class isCommunityAdmin extends PreExecutionRule {
     if (!USER || !fieldArgs?.input?.communityId) {
       return false;
     }
-    const user = await DB.query.communitySchema.findFirst({
-      with: {
-        usersToCommunities: {
-          where: (utc, { eq, and }) =>
-            and(eq(utc.userId, USER.id), eq(utc.role, "admin")),
-        },
-      },
+    const isCommunityAdmin = await DB.query.usersToCommunitiesSchema.findFirst({
+      where: (utc, { eq, and }) =>
+        and(
+          ...[
```

Comment:
el `...` no se si es necesario acá 🤔

Reply:
No afecta la lógica, pero estamos creando un array (cuando hacemos `[eq(), eq(), eq()]` y despues creando otro cuando le ponemos `...[]` nos podemos ahorrar crear 2 arrays por cada llamada

Reply:
@Benjvvp esto podria ser no un array, pero creo que podria ser un: 
`and(eq(), eq(), eq()),

Reply:
Es lo que hice debería estar listo en el último push